force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -32,7 +32,13 @@ public with sharing class VariablesDataTypesOperators {
      * @return The sum of the two numbers, or null if either number is null.
      */
     public static Integer addition(Integer a, Integer b) {  
-        return null; // Replace null with the variable you used to store the result
+        Integer sum1;
+        if (a == null || b == null) {
+            sum1 = null;
+        } else {
+            sum1 = a + b;
+        }
+        return sum1; // Replace null with the variable you used to store the result
```

Comment:
good. An easier way to write this might be 


public static Integer addition(Integer a, Integer b){
    if(a == null || b == null ){
        return null; // exit out early
    }
    return a + b; // if the values are null this will never be reached and makes it little easier to read in opinion. 
}